packages/create-next-app/templates/api/js/.env.example

```diff
@@ -0,0 +1,3 @@
+# Rename this file to `.env.local` to use environment variables locally with `next dev`
+# https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
+MY_HOST="example.com"
```

Comment:
We can likely remove this

Reply:
The other templates also share this file, should we still remove it?
x-ref: https://github.com/vercel/next.js/blob/canary/packages/create-next-app/templates/app/ts/.env.example